docs/partials/envvars.md

```diff
@@ -80,7 +80,7 @@ GITHUB_EXPORTER_WORKFLOWS_WINDOW
 : History window for querying workflows, defaults to `24h0m0s`
 
 GITHUB_EXPORTER_WORKFLOWS_LABELS
-: List of labels used for workflows, comma-separated list, defaults to `owner, repo, workflow, event, name, status, branch, number, run`
+: List of labels used for workflows, comma-separated list, defaults to `owner, repo, workflow, event, name, status, branch, number, run`. Optional labels: `actor`
```

Comment:
This will be reverted on the merge as this is automatically generated and currently there is no way to define optional values. I need to check a different way for the optional fields.